ivy/functional/frontends/tensorflow/math.py

```diff
@@ -307,6 +307,13 @@ def zero_fraction(value, name="zero_fraction"):
     return ivy.divide(count_zero, ivy.add(count_zero, count_nonzero))
 
 
+def argmin(input, axis, output_type=None, name=None):
+    if output_type in ["uint16", "int16", "int32", "int64"]:
+        return ivy.astype(ivy.argmin(input, axis=axis), output_type)
+    else:
+        return ivy.astype(ivy.argmin(input, axis=axis), "int64")
+
```

Comment:
The if statement you created is equivalent to just "return ivy.astype(ivy.argmin(input, axis=axis), output_type)". No need to check what the output_type is at all 🙂